runner/ansible/roles/2.2.2/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Set Test ID
```

Comment:
I don't know if we should have this check, that only passes if the SLE version is 15.
@lee-martin should we keep it?
Maybe we could check if the version is greater than 15 at most

Reply:
@arbulu89 This looks a little strange, but this is the first time I look at our Ansible checks, so I might be missing something.
So the original check ID 2.2.2 is in https://github.com/trento-project/trento/blob/main/examples/hana-scale-up-perf-optimized-azure.yaml  and that checks if the OS is 15 SP1 or newer (using the value 151).  To me this reads like somehow the test ID get mixed up with the target value we are actually checking for.

I also miss the "remediation" section, which most importantly contains the reason why we are even performing the check. Where does one find the remediation text in the ansible runner?

Reply:
> Hey @brett060102 ,
> Some new comments added.
> I'm not still sure about the 2.2 chapter.
> @diegoakechi @lee-martin @stefanotorresi Are we sure we want to include these checks now and here? Keep in mind that we cannot tune the expected values, so "constant" package and os versions will tend to fail in many cases.
> Opinions?

@arbulu89  I agree constant values are highly suspicious and should be reviewed. Also, all these checks seem to back the remediation text which references the reason why we are checking something. Where does one find the remediation text in the ansible runner? This would then justify why each check does what it does.
Looking at the where these checks were ported from ( https://github.com/trento-project/trento/blob/main/examples/hana-scale-up-perf-optimized-azure.yaml ) it seems many didn't have a reference and my guess is they originate from a spreadsheet we used early in Trento, but it did not have references.  We need to review all checks which do not have references anyway, and this process should then also uncover unrealistic checks, e.g. overly specific version checks of packages etc.  Can we open issues on a check by check basis to do this? I would not make this a stopper for implementing ainsible runner right now, we can fix these checks after the transition step by step.

Reply:
The remediation texts come in an additional file. We will add them in other PR. 
We just want to have here the check implementation.
The remediation will be copy/paste from the current bench common checks anyway

Reply:
My fault on this test. It actually checks if the version is **greater** than 151.
I guess we can move on with this by now